README.md

```diff
@@ -35,8 +35,41 @@ with intermezzOS, a learning kernel for the x86_64 platform, written in Rust
 
 ## Learning Objectives
 
+* Understand what an OS is.
+* Understand "boot to kmain" and why it has to happen.
+* Understand talking to hardware.
+* Understand some basic OS abstractions.
+
 ## Schedule
 
+The schedule is structured like a song, in ABABCB form. Some popular songs with
+this structure:
+
+* “What’s Love Got To Do With it” by Tina Turner
+* “Girl” by The Beatles
+* “Hot N Cold” by Katy Perry
+
+There's three kinds of sections: verses, choruses, and a bridge.
+
+* The chorus contains the main idea, or big picture, of what is being expressed
+  lyrically and musically. 
+* The verse contains the details of the song. Each verse is different.
+* The bridge is a device that is used to break up the repetitive pattern of the song.
+
+[for more](https://en.wikipedia.org/wiki/Song_structure)
+
```

Comment:
this all seems good except we need a environment setup part so the timing here all needs to shift. also as you already know, i think the timing here is Ambitious.